Address Methods comment 1.6 with sentence addition (#1031)
greenelab/covid19-review: content/60.methods.md

The COVID-19 Review Consortium provided a platform for researchers to engage in scientific investigation during the initial phase of the COVID-19 pandemic in 2020 at a time when many biological scientists were unable to access their research spaces.
In turn, by seeking to adapt Manubot to allow for broader participation in open publishing from fields where computational training in tools like version control is uncommon, we made a number of improvements that are expected to increase its appeal to researchers from all backgrounds.
Manubot provided a way for all contributors, including early-career researchers, to join a massive collaborative project, demonstrating their individual contributions to the larger work and gaining experience with version control.

> The licensing and infrastructure also provide a platform for individuals to adapt from this project to create their own snapshops of the COVID-19 literature that derive from, but are not wholly identical to, the master versions of these reviews.

Reviewer: ```suggestion
The licensing and infrastructure also provide an opportunity for individuals to adapt from this project to create their own snapshots of the COVID-19 literature that derive from, but are not wholly identical to, the primary versions of these reviews.
```
